src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('catan')
+        .directive('ctCardsPanel', [function() {
+
+            return {
+                restrict: 'E',
+                scope: {
+                    game: "="
+                },
+                templateUrl: "/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html",
+                link: function(scope) {
+                    scope.cards = function() {
+                        var currentGameUser = scope.game.getCurrentUser();
+                        return (currentGameUser) ? currentGameUser.developmentCards : {};
```

Comment:
why do we need this check? should be there any case when currentGameUser is undefined?